src/hx/files/File.hx

```diff
@@ -479,7 +479,19 @@ class File {
          return;
 
       #if (sys || macro || nodejs)
+         #if (flash || openfl)
+         //before writing a string, make sure path's parent exists.
+         //if it doesn't, it needs to be created to write the content for OpenFL and AIR/Flash.
+         var dir:String = haxe.io.Path.directory(path.toString());
+         if (!FileSystem.exists(dir))
+            FileSystem.createDirectory(dir);
+         var output = sys.io.File.write(path.toString(), false);
+         output.writeString(content);
+			output.close();
```

Comment:
Could `sys.io.File.saveContent(path.toString(), content);` be use instead after the directory was created or does `sys.io.File.write` need to be used?

Reply:
Tested sys.io.File.saveContent(path.toString(), content), but it did not work. I had to use File.write.

Reply:
ok, thanks